USAGE.md

```diff
@@ -0,0 +1,107 @@
+# Amazon Alexa Smart Home skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This skill allows you to connect your openHAB setup through the myopenHAB.org cloud service to Amazon Alexa.
+With this skill you can control items that respond to on/off and numeric commands as well as limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+# General configuration Instructions
+
+## Requirements
+
+* myopenHAB.org cloud service running on your instance of openHAB2
+* Amazon accout
+* Echo, Echo Dot, or another way to use Alexa (e.g. [Reverb](https://reverb.ai/))
+<!--- did I miss something? --->
+
+## 1. Skill Configuration
+
+1. Visit the [Alexa-Website](https://alexa.amazon.com/) or use the belonging app on your mobile.
+2. Navigate to "Smart Home" -> "Configure Smart Home" and search for openHAB.
+3. Follow the Instructions there to autohorise Alexa using your myopenhab.org-account.
+
+If you did not "tag" any items until now, first follow the steps below before you search for devices.
+(You can try, but you will not find any ;) )
```

Comment:
I'd remove this sentence for a serious documentation.

Reply:
Will do that while swapping 1 and 2